rasa/shared/utils/io.py

```diff
@@ -290,7 +290,7 @@ def json_to_string(obj: Any, **kwargs: Any) -> Text:
 def fix_yaml_loader() -> None:
     """Ensure that any string read by yaml is represented as unicode."""
 
-    def construct_yaml_str(self, node):
+    def construct_yaml_str(self: Any, node: Any) -> Any:
```

Comment:
no need to add typing for `self` 
```suggestion
    def construct_yaml_str(self, node: ScalarNode) -> Any:
```

Reply:
It actually requires one - otherwise when I run `make types` it fails on this last issue.

Reply:
interesting!